src/controllers/ioSignController.ts

```diff
@@ -137,21 +167,22 @@ export default class IoSignController {
   /**
    * Create a Signature from a Signature Request
    */
-  public readonly createSignature = (
-    req: express.Request
+  public readonly createSignature = async <T extends ResLocals>(
+    req: express.Request,
+    locals?: T
   ): Promise<
     | IResponseErrorInternal
     | IResponseErrorValidation
     | IResponseErrorNotFound
     | IResponseSuccessJson<SignatureDetailView>
   > =>
-    withUserFromRequest(req, async user =>
+    withUserFromRequest(req, user =>
       pipe(
-        req.body,
-        CreateSignatureBody.decode,
-        E.mapLeft(responseErrorValidation),
+        locals,
+        withLollipopLocals,
```

Comment:
can you add a comment on what's happening here?
(what are "lollipop locals"?)

Reply:
`withLollipopLocals` is an utility function of lollipop domain. Its role is described in the [comment](https://github.com/pagopa/io-backend/blob/8a92889e92d71877e7f4a6d29b77e42899828e93/src/types/lollipop.ts#L54) of the function.

Reply:
it's still hard to figure it out what's happening here despite that comment

Reply:
I add a comment!